app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for create is too high. [20.32/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)